.github/workflows/cronJob.yaml

```diff
@@ -74,16 +74,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # Existing LTI release tags and branches can be added to obtain build results.If the tag array is empty, it will default to 'main'. However, if there is at least one tag or branch in the tag array and you need to run on 'main' as well, make sure to add 'main' to the array.
+        tag: [ 'lsp4ij-market-0.0.2-integration' ] # Can specify tags or branches such as '24.0.6' and 'main'
```

Comment:
@TrevCraw While merging this change to main, we have to update the value in the tag. So, are you planning to keep the issue open till we merge these changes to main?

Reply:
@aparnamichael , @TrevCraw suggested me to add a check box to this Epic: https://github.com/OpenLiberty/liberty-tools-intellij/issues/666

So I have added in this epic.